client/src/app/tabs/cloud-bpmn/BpmnEditor.js

```diff
@@ -757,7 +822,8 @@ export class BpmnEditor extends CachedComponent {
 
     const modeler = new BpmnModeler({
       ...options,
-      position: 'absolute'
+      position: 'absolute',
+      changeTemplateCommand: 'propertiesPanel.zeebe.changeTemplate'
```

Comment:
I don't quite get why we need to have two separate commands for template changes. Shouldn't it be that I have either Zeebe or Camunda Platform element templates but never both of them in one project?

Reply:
Actually, that was a decision to get the `camunda` namespace out of zeebe land. The other choice would be to break the existing element templates by using a unified command. To avoid this, we simply added a similar one for zeebe.

https://github.com/bpmn-io/bpmn-js-properties-panel/pull/555

If we want to change this, we would have to roll back it in the properties panel.